### ISSUE TITLE
Image data between 0 - 255 and not between 0 - 1

### ISSUE THREAD

**Issue by XMaster96** [hill-a/stable-baselines#328]
When using a image env, the image data that gets fed in to the neural network is between 0 and 255, should this not be between 0 and 1 so that the neural network works better?

**Comment by araffin** [comment]
Hello,

I thought this was documented somewhere but I could only find an obscure mention [here](https://stable-baselines.readthedocs.io/en/master/guide/examples.html#mujoco-normalizing-input-features).

So, to answer your question, this is done automatically here: https://github.com/hill-a/stable-baselines/blob/c9be8dcddcf126f1ce315e6578ce96fddc5613d0/stable_baselines/common/input.py#L32-L33


I think we need to update the documentation, so that becomes explicit.

**Comment by XMaster96** [comment]
The we have a couple of bug....
And I was starting to believe that this could be a feature, considering how consistently it does not work. 
let me explain:

The first bug is in `dataset.py`. I realised that the images get at no point normalized, in the image loading chain.
You can test that by simply printing the data from the image loader https://github.com/hill-a/stable-baselines/blob/fddf169875154f6129071045f0a6f99614c490a5/stable_baselines/common/base_class.py#L227 and run `test_pretrain_images()` from test_gail. 
I propose that I simplify fix that with my current PR.



I looked for the second one at a2c.
For that I print the `obs` data that flow through the model:
https://github.com/hill-a/stable-baselines/blob/fddf169875154f6129071045f0a6f99614c490a5/stable_baselines/a2c/a2c.py#L173
and run this code

```
from stable_baselines import A2C
from stable_baselines.common.cmd_util import make_atari_env
from stable_baselines.common.vec_env import VecFrameStack

env = make_atari_env("PongNoFrameskip-v4", num_env=1, seed=0)
env = VecFrameStack(env, n_stack=4)
model = A2C('CnnPolicy', env)
model.learn(total_timesteps=300)
```

unfortunately the printed `obs` are in between 0 and 255.

**Comment by araffin** [comment]
Hello,

I disagree for the two bugs, the thing is `observation_ph` is the unnormalized input placeholder, but this is `processed_obs` (the normalized placeholder) that is used as input to the CNN:

https://github.com/hill-a/stable-baselines/blob/c9be8dcddcf126f1ce315e6578ce96fddc5613d0/stable_baselines/common/policies.py#L559

So that's normal that you see only unnormalized obs, the normalization is done by tensorflow when computing the value of `processed_obs` given `observation_ph`.

**Comment by XMaster96** [comment]
oh ok, I was not aware of that.
so quotation solved thanks.
